deepchem/models/torch_models/layers.py

```diff
@@ -14,19 +16,19 @@ class ScaleNorm(nn.Module):
   Finally, the result is returned as `input_tensor * norm value`.
 
   This layer can be used instead of LayerNorm when a scaled version of the norm is required.
-  Instead of performing the scaling operation (scale / norm) in a lambda-like layer, we are defining it within this layer to make prototyping more efficient.
+  Instead of performing the scaling operation (`scale / norm`) in a lambda-like layer, we are defining it within this layer to make prototyping more efficient.
 
   References
   ----------
   .. [1] Lukasz Maziarka et al. "Molecule Attention Transformer" Graph Representation Learning workshop and Machine Learning and the Physical Sciences workshop at NeurIPS 2019. 2020. https://arxiv.org/abs/2002.08264
-  
+
   Examples
   --------
   >>> import deepchem as dc
   >>> scale = 0.35
-  >>> layer = dc.models.torch_models.ScaleNorm(scale)
-  >>> input_tensor = torch.tensor([[1.269, 39.36], [0.00918, -9.12]])
-  >>> output_tensor = layer.forward(input_tensor)
+  >>> layer = dc.models.torch_models.layers.ScaleNorm(scale)
+  >>> input_tensor = torch.Tensor([[1.269, 39.36], [0.00918, -9.12]])
```

Comment:
`torch.tensor` is the function making the tensor, `torch.Tensor` is the type

Reply:
Done!